Update frontier_silicon documentation (#21238)
home-assistant/home-assistant.github.io: source/_integrations/frontier_silicon.markdown

ha_category:
  - Media Player
ha_iot_class: Local Push

> ha_release: '2022.1'

Reviewer: This version marks the introduction (not the last update).
Please revert this change.

ha_domain: frontier_silicon
ha_platforms:
  - media_player
ha_codeowners:
  - '@wlcrs'
ha_ssdp: true
ha_config_flow: true
---

This integration provides support for Internet Radios based on the [Frontier Silicon chipset]. Some of the manufacturers which offer products based on these chips include: Hama, Medion, Slivercrest, Auna, Technisat, Revo, Pinnel, etc. These devices will be usually controlled by the [UNDOK] app.

> {% include integrations/config_flow.md %}

Reviewer: I think this makes more sense to add after the prerequisites?

Reviewer: So maybe you misunderstood me? Thus unresolving.

I think the order should be:

- Supported models
- Prerequiresites
- Configuration

Author: Should be ok now, sorry!

> ## Supported Models
> Frontier Silicon is used by many different brands of radio manufacturers.

Reviewer: ```suggestion
## Supported Models

Frontier Silicon is used by many different brands of radio manufacturers.
```

Reviewer: Unresolved, as it was not addressed.

## Supported Models
Frontier Silicon is used by many different brands of radio manufacturers.

Supported devices include, but are not limited to:

* Hama: [IR50], [IR110], [DIR3110]
* Medion: [Medion Radios]
* Silvercrest: [SIRD 14 C2 (archived website)]
* Teufel: [Radio 3sixty (2019)]
* Roberts: [Roberts Stream 94i]
* Some models from: Auna, Technisat, Revo, Pinell, Como Audio

This integration was developed and tested with a [Roberts Stream 94i].

If your device is supported by the UNDOK app, then it is also supported by this integration.

## Prerequisites

The integration supports automatic discovery of your Internet Radio. If you need to set up the device manually, please provide the device IP-address. Some models use a separate port (2244) for API access, this can be verified by visiting `http://[host]:[port]/device`.

The default PIN for Frontier Silicon-based devices is 1234. You can set the PIN code of your device (depending on manufacturer) under:
*MENU button > Main Menu > System setting > Network > NetRemote PIN setup*

> In case your device (friendly name) is called *badezimmer*, an example automation can look something like this:

Reviewer: Let's move the example us using it, under the config flow procedures.

* Medion: [Medion Radios]
* Silvercrest: [SIRD 14 C2]
* Teufel: [Radio 3sixty(2019)]
* Roberts: [Roberts Stream 94i]
* Some models from: Auna, Technisat, Revo, Pinell, Como Audio

This integration was developed and tested with a [Roberts Stream 94i].

> ## Configuration

Reviewer: ```suggestion
## Prerequisites
```

> The integration supports automatic discovery of your Internet Radio. If you need to setup the device manually, please provide the device URL, which is typically something like `http://[host]/device`. Some models use a separate port (2244) for API access, this can be verified by visiting http://[host]:[port]/device.

Reviewer: ```suggestion
The integration supports automatic discovery of your Internet Radio. If you need to set up the device manually, please provide the device URL, which is typically something like `http://[host]/device`. Some models use a separate port (2244) for API access, this can be verified by visiting `http://[host]:[port]/device`.
```

## Development

Support is provided through the Python [afsapi] module. The Python module was developed by using the documentation provided by [flammy] and

> is based on [tiwillam]'s fsapi project. Special thanks to both developers, this integration would have not been possible without their work.

Reviewer: I think this section just needs to go. It isn't relevant for the end-user of Home Assistant.

## Notes and Limitations

> <div class='note warning' name="fsapi-session-note">

Reviewer: ```suggestion
<div class='note warning'>
```
